vllm/utils.py

```diff
@@ -304,6 +304,14 @@ def is_hip() -> bool:
     return torch.version.hip is not None
 
 
+def is_flashinfer() -> bool:
+    try:
+        import flashinfer
+    except ImportError:
+        flashinfer = None
+    return flashinfer is not None
```

Comment:
We should not use this function to determine whether to use FlashInfer. This serious of functions are mainly for backend devices while FlashInfer is one of attention backends